### ISSUE TITLE
site redesign & rebrand

### ISSUE THREAD

**Issue by boneskull** [mochajs/admin#12]
some time ago we discussed (purchasing) a site redesign and rebranding. I think we should take steps to move forward with this; we can use the OJSF's resources to get a vetted contractor to work on the project. 

We need to answer a couple questions first. 

1. What is our budget?
1. What will be the scope of the work?

I'll give my opinion:

1. Budget: we have ~40k USD banked as of this writing. I do not know what is reasonable for a redesign of our magnitude, but I would not want to spend _all_ of it. It's also difficult for me to justify spending a huge sum on something that is, in my mind, _wanted_ but not _needed_ not _critical_. Just throwing a number out there: maybe 10k?  
1. Scope:  Ideally, we would have someone redesign and rebrand and implement everything so we don't need to lift a finger. But that's not going to happen; someone would need to integrate this into 11ty and the JsDoc template engine.  This may be something we want to pay for, but perhaps integration is not the same contractor. anyway, I'd like to see:
    - A new logo, alternate and text mark
    - A design system: widgets, interactions, styles, and/or components that we can use
    - A site layout: our site is split into two main areas; a very large landing page containing end-user docs, and another section for API docs (which is many pages).  I'd like everything to be more tightly integrated, organized, and easily searched, but I _would_ like to abandon the our "huge single page" concept. Splitting things up should help readers focus on finding the information they need while ignoring what they do not. 
    - Deliverables would be in some format which we can easily export into images, styles, tags/components, SVGs, etc, such as a Sketch file.  Of these, I'm unsure about styles (CSS) and markup: does it need to be hand-written?

Regarding the file format, another possibility may be Figma, but I'm unfamiliar with it. Sketch is not free, so we may want to purchase ourselves licenses (I already have a license) if we wish to work with it. 

I suggest we can talk about implementation in a separate discussion; again, I think it makes sense to pay for this, but it may need to be a separate project. Please let me know if you disagree, but IMO it will be difficult to find a single person to do the design work _and_ a complete implementation. 

Hopefully you can respond sometime this weekend, as I'd like to get going on this early next week.  

cc @mochajs/core @jorydotcom

**Comment by outsideris** [comment]
Agreed. Currently, our site is too long. So, we need to make a new design. 
I didn't think about rebranding. But, we can delegate a desiger to be able to make a new brand than a designer is forced to fit colors or designs to our current logo.

**Comment by Munter** [comment]
Thumbs up on all points. Especially the one about splitting the single page up into a more coherent informational hierarchy across multiple pages

**Comment by boneskull** [comment]
What of the budget?

**Comment by Munter** [comment]
10k sounds good, but I don't know what you can get for that. I'd say no at 20k

**Comment by boneskull** [comment]
ok. so "under 20k" sounds reasonable. I am really unsure how much work this is going to be for someone, but I am also aware it's just not as valuable to us.

**Comment by boneskull** [comment]
notes:

- we want a design _language_ not necessarily a _system_. we don't need a _system_. we have no real _widgets_
- "sitemap": information architecture
- two contractors; one for the logo/"brand" and another for IA & design lang

**Comment by outsideris** [comment]
How can we find contractors? 

I know a designer. I'm not sure if we will satisfy the designer's result. But I can help to integrate out 11ty and JSdoc templates with the design if the designer work with us.
